scripts/ci-docker-images/verifier-ubuntu-20-04/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
This is technically only needed for 20.04 which is pulling in tzdata as a dependency, but I figured I'd add it to them all.